crypto/bls/aggregation.go

```diff
@@ -134,7 +134,7 @@ func (a *StdSignatureAggregator) AggregateSignatures(
 				aggSigs[ind] = &Signature{sig.Deserialize(sig.Serialize())}
 				aggPubKeys[ind] = op.PubkeyG2.Deserialize(op.PubkeyG2.Serialize())
 			} else {
-				aggSigs[ind].Add(sig.G1Point)
+				aggSigs[ind].Add(sig)
```

Comment:
I am curious why has this changed?

Reply:
Because Signatures wrap a G1Point, and the library originally didn't overwrite the Add method, so that add method being used was signatures was actually the Add method of the underlying G1Point. I found it ugly that you needed to do `sig.Add(othersig.G1point)`, so added the method on signatures directly so that you just have to do `sig.Add(otherSig)`